docs/pages/database-access/troubleshooting.mdx

```diff
@@ -137,3 +137,18 @@ Now Alice can connect to any database in the Teleport cluster using any database
 This example is intentionally simple; we could have configured Alice's permissions using more fine-grained control.
 For more detailed information about database access controls and how to restrict
 access see the [RBAC](../database-access/rbac.mdx) documentation.
+
+## Connection to MySQL database results in "Unknown system variable 'query_cache_size'" error
+When a Teleport Proxy is configured with Teleport Proxy Listener Mode (`proxy_listener_mode: separate`) by default MySQL client connections to the Teleport Proxy don't use the TLS Routing feature that allows propagating correct MySQL Server Version.
+
+By default, Teleport Proxy returns `8.0.0-Teleport` as the MySQL Server version. When the default MySQL Version is returned. In some cases like connecting with GUI Client, this can result in obtaining an `Unknown system variable 'query_cache_size'` error that indicates that MySQL Capabilities were not properly negotiated between MySQL Client and Server.
```

Comment:
```suggestion
By default, the Teleport Proxy Service returns `8.0.0-Teleport` as the MySQL server version. In some cases, like connecting with a GUI Client, this can result in obtaining an `Unknown system variable 'query_cache_size'` error that indicates that MySQL capabilities were not properly negotiated between the MySQL client and server.
```